### ISSUE TITLE
hit@10在LP任务的testing阶段一直为0

### ISSUE THREAD

**Issue by yuanXuX** [yao8839836/kg_bert#26]
Hi, 你好, 非常感谢你结合kg&bert做link prediction的相关工作.

1. 想请问按照你提供的运行命令, 在FB15k237数据集的实验中,test阶段 一直hit@10 until now:  0.0, 但是MR缓步下降是什么原因造成的呢?

2. 其中,WARNING - pytorch_pretrained_bert.optimization -   Training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule correctly.是什么原因呢?
